### ISSUE TITLE
how to commit with --no-verify option

### ISSUE THREAD

**Issue by SmithJackyson** [ForkIssues/Tracker#1970]

**Comment by DanPristupov** [comment]
https://github.com/fork-dev/Tracker/issues/948#issuecomment-604000807

**Comment by SmithJackyson** [comment]
> #948 (comment)

Is there another way to skip pre-commit hook before commit, you fixed it, but you have to run pre-commit hook when recommit, that will take some time.

**Comment by DanPristupov** [comment]
No, I don't know where to put this option. It need to be intuitive but also not bother people who don't use hooks.

**Comment by SmithJackyson** [comment]
> No, I don't know where to put this option. It need to be intuitive but also not bother people who don't use hooks.

<img width="721" alt="截屏2023-09-16 11 11 45" src="https://github.com/fork-dev/Tracker/assets/11938336/54427978-6f96-43ba-94d5-6236a65fc36e">

this is my advice, you can design a button on commit view, this button will show secondary menu, put some rarely used options in this menu. people use those options will know what they are doing, and it will not bother people who don't use hooks

**Comment by ValentinGurkov** [comment]
I can also suggest adding the option to the "commit options" dropdown on the right.

**Comment by Tensho** [comment]
Could we have this as UI element please? Sometimes I don't want to wait for all hooks to be completed.

**Comment by christos** [comment]
A more subtle way to implement this would be to simply change the button when `option` is held down to read `Commit 2 files, skipping pre-commit hooks` — Kinda of like how menubar items change when `option` is held down. This would simply change `Commit` to `Commit, no verify` in the `Repository` menu and the button

**Comment by mveroukis** [comment]
There's no reason for this to be subtle. It's far more important to have a Git client that doesn't force us to use other tools or the command line. Just add a checkbox next to Amend called No Verify and be done with it. You're over thinking this.

I'd get rid of the Option + click to Commit & Push and add that as a checkbox as well. Or perhaps it's own button. These undiscoverable UI patterns are annoying.

**Comment by 343max** [comment]
I really need this feature on a regular basis and the absence of it is driving me away from fork which is a shame. Don't let the perfect be the enemy of the good and just add the checkbox next to "amend" or some other quick solution.
